sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py

```diff
@@ -74,7 +74,8 @@ def create_certificate(
         :param bool enabled: Determines whether the object is enabled.
         :param tags: Application specific metadata in the form of key-value pairs.
         :type tags: dict(str, str)
-        :returns: The created CertificateOperation
+        :returns: An LROPoller for the create certificate operation. Waiting on the poller
```

Comment:
I'd link to `LROPoller`, `Certificate`, and `CertificateOperation` here.

Reply:
switched over too, thanks!